custom/crediblex/portfolio/loanaccount/src/main/java/com/crediblex/fineract/portfolio/loanaccount/service/CustomLoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -113,6 +117,14 @@ public CustomLoanWritePlatformServiceJpaRepositoryImpl(PlatformSecurityContext c
     @Override
     public CommandProcessingResult disburseLoan(final Long loanId, final JsonCommand command, Boolean isAccountTransfer,
                                                 Boolean isWithoutAutoPayment) {
+        // Line of Credit validation (before standard validation to catch LOC-specific errors first)
+        try {
+            locLoanApplicationValidator.validateLineOfCredit(command.parsedJson());
+        } catch (Exception e) {
```

Comment:
You should not be catching and rethrowing this exception that should be thrown right from the validation code